icons/icons.json

```diff
@@ -2576,16 +2576,20 @@
 			}
 		]
 	},
-	"file_type_python": {
+	"file_type_pypi": {
 		"aliases": [
 			{
 				"extensions": [
-					".python-version"
+					"requirements.in",
+					"requirements.txt"
 				],
-				"name": "Plain Text (Python)",
-				"scope": "text.plain.python"
```

Comment:
At least https://packagecontrol.io/packages/requirementstxt provides a real syntax for this file type. So maybe adjust declaration accordingly.

Also I am not quite sure requirements.txt is directly related with pypi. It's rather a way to list dependencies to be installed via pip.

see: https://pip.pypa.io/en/stable/reference/requirements-file-format/

Reply:
I didn't know about that package - adjusted the definition to use `source.requirementstxt`.

Re: PyPI - what I've aimed for here is consistency with JS filetypes - there, `package.json` is under the `npm` umbrella even though it's not tied to any package manager or package registry (or even runtime - even Deno takes `package.json` into consideration nowadays).

Changed to `file_type_pip` which I've probably should've done from the start.